### ISSUE TITLE
Add property similar to "description" to the DB wrapper

### ISSUE THREAD

**Issue by Cito** [PyGreSQL/PyGreSQL#46]
In #39 we added a method `fieldinfo()` to the query in the classic module, which takes an optional field name or number. If nothing is specified, the field info for *all* fields of the query is returned.

This method is a bit low-level though, the provided information is pretty internal and difficult to interpret. On the `DB` wrapper class, we may want to add another higher-level method or property similar to the `description` attribute in the DB-API 2 module. This could be a little bit more informative and PostgreSQL-specific than the DB-API 2 `description`, but the idea would be the same. We could even reuse much of the code of the `description` property in the DB-API 2 since it is also based on a lower-level method similar to `fieldinfo()`.

**Comment by mansi104-ai** [comment]
hi , @Cito , Is this issue still open ?
Can you elaborate it ?

**Comment by Cito** [comment]
Sure, this is still open. Will do a triage of issues before releasing 6.2. Feel free to comment and issues, upvote them, make suggestions or provide PRs.

**Comment by mansi104-ai** [comment]
how can I contribute ?

**Comment by Cito** [comment]
By doing any of the things above. Or by going away if you're just an AI bot.

**Comment by justinpryzby** [comment]
I don't think this is even possible with the current design.
    In pgdb, you can do c=pgdb.connect().cursor() c.execute(); c.description.
    But in pg, the corresponding thing would be c=pg.DB(); d=c.query();
    The DB object doesn't reference the query result.  It'd need to be in
    the pgquery.c query object, which is where we put fieldinfo.  So it
    wouldn't be higher-level.

**Comment by Cito** [comment]
That's true - we would need to add it to the pgquery object (but then we can't use Python) or add a method to DB that takes the query or its fieldinfo as a parameter.
